src/components/structures/MessagePanel.tsx

```diff
@@ -608,6 +613,11 @@ export default class MessagePanel extends React.Component<IProps, IState> {
         }
     }
 
+    private isSentState(ev: MatrixEvent): boolean {
+        const status = ev.getAssociatedStatus();
+        return !status || status === EventStatus.SENT;
```

Comment:
Why is a falsy status considered as a sent state (function name)?
Can you add a comment?

Reply:
![image](https://github.com/matrix-org/matrix-react-sdk/assets/2403652/e32c9096-911a-4157-a7b2-74f2ee794e6b)

That's just how MatrixEvent works, the status is only ever of outgoing events, and once we get the remote-echo we clear the status